solutions/standard/variables.tf

```diff
@@ -0,0 +1,657 @@
+##############################################################################
+# Common
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API key used to provision resources."
+  sensitive   = true
+}
+
+variable "existing_secrets_manager_crn" {
+  type        = string
+  description = "CRN of secrets manager instance to create the secret engine in"
+  nullable    = false
+}
+
+variable "region" {
```

Comment:
this input is not needed. The region can be parsed from the `existing_secrets_manager_crn` input